### DISCUSSION TITLE
How to authenticate when using the API?

### DISCUSSION THREAD

**Discussion by colbach** [hay-kot/homebox#795] [Q&A]
Hello, I am trying to play around with the API (basically I want to create a two way synchronization for my home made inventory system).

The API documentation says "API Key: Bearer". Where can I find the API key? I didn't see an option in the UI to reveal it.

**Comment by colbach** [comment]
If anyone is wondering. I found it out myself. Just log in via the web interface, open inspect element and then go to network. Click something in the web interface triggering an API request, then click on the request, go to Cookies and copy hb.auth.token

**Reply by mtmustski** [reply]
Thanks, went through those same motions myself over the weekend trying to authenticate via the API. It would be awesome if we could generate an authentication token or something on the profile page. That way we could have a longer lasting token / secret to be used for accessing the API.

**Reply by salsaw** [reply]
You can also use the curl command or a tool like Postman to generate the token. Just pass your username and password to the API.
